invokeai/app/invocations/baseinvocation.py

```diff
@@ -807,9 +811,9 @@ def wrapper(cls: Type[TBaseInvocation]) -> Type[TBaseInvocation]:
         cls.UIConfig.category = category
 
         # Grab the node pack's name from the module name, if it's a custom node
-        module_name = cls.__module__.split(".")[0]
-        if module_name.endswith(CUSTOM_NODE_PACK_SUFFIX):
-            cls.UIConfig.node_pack = module_name.split(CUSTOM_NODE_PACK_SUFFIX)[0]
+        is_custom_node = cls.__module__.rsplit(".", 1)[0] == "invokeai.app.invocations"
+        if is_custom_node:
+            cls.UIConfig.node_pack = cls.__module__.split(".")[0]
```

Comment:
This change was unrelated to the PR, but it's small so I'm leaving it here. Makes checking if nodes are custom or core simpler. Also see the change in `custom_nodes/init.py`.